__tests__/e2e/wix-sdk-e2e.latest-versions.spec.ts

```diff
@@ -0,0 +1,92 @@
+import tmp from 'tmp';
+import { bundle } from '../../src';
+import fs from 'fs';
+import { getLatestVersionName, validateTypescript } from './utils';
+
+describe('Latest versions', () => {
```

Comment:
Why this is called `Latests versions` ? 🤔 
The tests validate asterisks exports, etc.